python/django-ninja/config.yaml

```diff
@@ -0,0 +1,12 @@
+framework:
+  website: http://django-ninja.rest-framework.com/
+  version: 0.3.1
```

Comment:
could you use `X.Y` format ? e.g : `0.3`

Reply:
done